yogstation/code/modules/uplink/uplink_item.dm

```diff
@@ -116,3 +116,10 @@
 	item = /obj/item/melee/fryingpan/bananium
 	cost = 40
 	cant_discount = TRUE
+
+/datum/uplink_item/stealthy_weapons/camera_flash
+	name = "Camera flash"
+	desc = "A camera with an upgraded flash. Can be used much like a flash except with a much longer cooldown between uses."
+	item = /obj/item/camera/syndicate
+	cost = 4
+	surplus = 20
```

Comment:
surplus cost 20?

Reply:
> surplus cost 20?

That's what it says; yeah

Reply:
oh that's for the surplus crate not discount price.